src/main/java/org/wiremock/grpc/dsl/WireMockGrpcService.java

```diff
@@ -51,4 +53,23 @@ public GrpcVerification verify(int count, String method) {
   public GrpcVerification verify(CountMatchingStrategy countMatch, String method) {
     return new GrpcVerification(wireMock, countMatch, serviceName, method);
   }
+
+  /** Resets stubs for the current gRPC service */
+  public void resetStubs() {
```

Comment:
Can we rename this to `removeAllStubs()`? I think this will more clearly distinguish from WireMock's concept of resetting where the persistent stubs on the filesystem will be reloaded.

Reply:
Done